src/lib/include/xtgeo/regsurf.hpp

```diff
@@ -59,8 +63,7 @@ sample_grid3d_layer(const size_t ncol,
                     const py::array_t<float> &zcornsv,
                     const py::array_t<int> &actnumsv,
                     const size_t klayer,
-                    const int option,
-                    const int activeonly);
+                    const int num_threads);
```

Comment:
Needed to have as arguments, as the tests (in Python) did not capture a change in env variable `OMP_NUM_THREADS"

Reply:
I'm guessing the environment variable would have to be set before `pytest` is invoked and xtgeo is loaded into memory. Another option could be to benchmark from a script subprocessed with that environment variable set, but that'd be a bit annoying to implement